.github/workflows/acceptance-tests-runner.yml

```diff
@@ -382,6 +382,38 @@ jobs:
             ./internal/service/advancedclustertpf
         run: make testacc
 
+  advanced_cluster_tpf_move:
+    needs: [ change-detection, get-provider-version ]
+    if: ${{ (needs.change-detection.outputs.advanced_cluster_tpf == 'true' || inputs.test_group == 'advanced_cluster_tpf') && inputs.terraform_version == '1.10.x' }}
+    env:
+      MONGODB_ATLAS_ADVANCED_CLUSTER_V2_SCHEMA: 'true'
+    strategy:
+      max-parallel: 2
+      fail-fast: false
+      matrix:
+        terraform_version: ['1.8.x', '1.9.x'] # TF supported versions for MoveState apart from latest one, 1.10.x
```

Comment:
I guess this must be updated manually when a new TF version is released?

Reply:
yes, but I would expect these checks to be temporary while we're developing the functionality.  At some moment we'll be able to remove these specific checks for MoveState